e2e/pages/adf/process_services/processListDemoPage.ts

```diff
@@ -0,0 +1,153 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Util } from '../../../util/util';
+import { ContentListPage } from '../dialog/contentListPage';
+import { element, by, protractor } from 'protractor';
+
+export class ProcessListDemoPage {
+
+    appIdInput = element(by.css('input[data-automation-id="app-id"]'));
+    resetButton = element(by.cssContainingText('button span', 'Reset'));
+    emptyProcessContent = element(by.css('div[class="adf-empty-content"]'));
+    processDefinitionInput = element(by.css('input[data-automation-id="process-definition-id"]'));
+    processInstanceInput = element(by.css('input[data-automation-id="process-instance-id"]'));
+    stateSelector = element(by.css('mat-select[data-automation-id="state"'));
+    sortSelector = element(by.css('mat-select[data-automation-id="sort"'));
+
+    contentList = new ContentListPage();
+
+    async getAppDefinitionId(alfrescoJsApi, appModelId) {
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
```

Comment:
This method should be moved, cause it's not related to a value of an element on the page, just from a response of a call.